modules/distr-tracing-deploy-streaming.adoc

```diff
@@ -0,0 +1,153 @@
+////
+This module included in the following assemblies:
+- distr_tracing_install/distr-tracing-deploying.adoc
+////
+
+[id="distr-tracing-deploy-streaming_{context}"]
+= Deploying the {DTShortName} streaming strategy from the web console
+
+The `streaming` deployment strategy is intended for production environments, where long term storage of trace data is important, as well as a more scalable and highly available architecture is required.
+
+The `streaming` strategy provides a streaming capability that sits between the Collector and the storage (Elasticsearch). This reduces the pressure on the storage under high load situations, and enables other trace post-processing capabilities to tap into the real time span data directly from the streaming platform (Kafka).
```

Comment:
Actually I think it's "real-time span data" since distributed tracing collects "spans".  I can't use your second suggestion because my copy of IBM Style says hyphen for the adjective (real-time data), no hyphen for the noun (in real time).  (Bah, why is writing so complicated sometimes!!!)